controllers/manila_controller.go

```diff
@@ -989,6 +989,7 @@ func (r *ManilaReconciler) generateServiceConfig(
 		return err
 	}
 	transportURLSecretData := string(transportURLSecret.Data["transport_url"])
+	transportURLQuorumQueues := string(transportURLSecret.Data["quorumqueues"]) == "true"
```

Comment:
good, so this is always set to `false` unless it is set to `true` within `transportURLSecret`. I guess this is fair enough and provides backward compatibility.